SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestCrossServerRPC/SpatialTestCrossServerRPC.cpp

```diff
@@ -12,7 +13,19 @@
  * server-to-server RPCs. The test includes 4 server workers and 2 clients. NOTE: This test requires the map it runs in to have the
  * BP_QuadrantLBStrategy and OwnershipLockingPolicy set in order to be relevant.
  *
- * The flow is as follows:
+ * The tests are in two sections: first startup actor RPC tests and then dynamic actor RPC tests.
+ *
+ * The flow for the startup actor tests is as follows:
+ * - Setup:
+ *  - The level contains one CrossServerRPCCube on Server 4 that is not replicated initially.
+ *  - On authoritative server we turn on replication, change the authority to non-authoritative and then send an RPC. These specific steps
```

Comment:
```suggestion
 *  - On all non-authoritative servers we turn on replication, explicitly set authority to non-authoritative and then send an RPC. These specific steps
```

Reply:
Worth adding that because the cube is initially not replicated, it's locally marked as authoritative on all servers, even though only server 4 should have authority. This is a quirk of `UGlobalStateManager::HandleActorBasedOnLoadBalancer`